### DISCUSSION TITLE
Is it possible to use this package to create a  LSTM VAE

### DISCUSSION THREAD

**Discussion by osbm** [clementchadebec/benchmark_VAE#39] [Q&A] [answered]
Let's say I want to use encode SMILES to latent space. Is it possible to use a sequential algorithm with this package, if so could you add a very simple example to the examples folder?

Any help is appreciated. :)

**Comment by clementchadebec** [comment, marked as answer]
Hi @osbm,

Thank you for opening this discussion! I will try to work on a simple example in the next few days and add it to the tutorials :)

Best,

Clément

**Reply by osbm** [reply]
you are awesome. i wanted to make an LSTM ae from scratch but finding this project will save me a lot of time
